src/lib/services/submit.services.ts

```diff
@@ -225,7 +225,23 @@ export const submitAddNodeProviderProposal = async ({
 		ProposalParams,
 		'neuronId'
 	>): Promise<SubmitProposalResult> => {
-		const { title, url, nodeProviderId, summary } = metadata;
+		const { url, nodeProviderId } = metadata;
+
+		const title = 'Add Node Provider: ' + metadata?.nodeProviderName;
```

Comment:
You are going to provide some validation or you expect the backend to bubble human readable errors if the node provider is not provided?

Reply:
So for the `url` and `nodeProviderId` there is already validation via the NNS. For the other fields: `nodeProviderName`, `urlSelfDeclaration`, `urlIdentityProof`, `hashSelfDeclaration`, `hashIdentityProof` -- these will have validation in a separate PR. 

I was thinking to address this in a PR that also solves issue #19. Wdyt?

Reply:
Ok